cmd/swapcli/main.go

```diff
@@ -643,13 +657,16 @@ func runGetPastSwap(ctx *cli.Context) error {
 		return err
 	}
 
-	fmt.Printf("Provided: %s\n ProvidedAmount: %v\n ReceivedAmount: %v\n ExchangeRate: %v\n Status: %s\n",
-		info.Provided,
-		info.ProvidedAmount,
-		info.ReceivedAmount,
-		info.ExchangeRate,
-		info.Status,
-	)
+	receivedCoin := "ETH"
```

Comment:
it might not be ETH necessarily, check the info for the asset instead?

Reply:
also if we're a maker this will be XMR

Reply:
That is what I tried to do first, but info's type doesn't have the needed fields:
```
type GetPastResponse struct {
	Provided       coins.ProvidesCoin  `json:"provided"`
	ProvidedAmount *apd.Decimal        `json:"providedAmount"`
	ReceivedAmount *apd.Decimal        `json:"expectedAmount"`
	ExchangeRate   *coins.ExchangeRate `json:"exchangeRate"`
	Status         string              `json:"status"`
}
```
We'll have to fix it in a future PR.